aggregator.py

```diff
@@ -605,6 +617,10 @@ def __init__(self, hostname, interval=1.0, expiry_seconds=300, formatter=None, r
             'ms': Histogram,
             's': Set,
         }
+        self.histogram_aggregates = histogram_aggregates\
+           or get_histogram_aggregates()
```

Comment:
Maybe it's unclear I guess, this is to provide a sane default (calling get with no args) when people instantiate an aggregator without getting from config a anything.